### ISSUE TITLE
import doesn't work

### ISSUE THREAD

**Issue by marvindao** [dask/s3fs#478]
**What happened**: 
Error when `import s3fs`:
`ImportError: cannot import name 'setup_logging' from 'fsspec.utils'`

The bug in core.py: `from fsspec.utils import infer_storage_options, tokenize, setup_logging as setup_logger` where setup_logging is no longer in fsspec.utils. it's already in core.py

**What you expected to happen**:

**Minimal Complete Verifiable Example**:

```python
`import s3fs`
```

**Anything else we need to know?**:

**Environment**:

- Dask version: s3fs version 2021.4.0, fsspec 2021.4.0
- Python version: 3.7.6
- Operating System: Windows
- Install method (conda, pip, source): pip

**Comment by martindurant** [comment]
Are you certain you have those versions?
```
$ conda create -n blah python=3.7 ipython
$ conda activate blah
$ pip install s3fs==2021.4.0 fsspec==2021.4.0
$ python
>>> import s3fs
```
works fine on my system

**Comment by marvindao** [comment]
hmm this's what I see on my system

![image](https://user-images.githubusercontent.com/54801762/117857763-0b01e480-b285-11eb-8d2b-bf9fee11b7d8.png)


Do you see the following line in s3fs/core.py:
![image](https://user-images.githubusercontent.com/54801762/117857986-4dc3bc80-b285-11eb-825f-57fde0c0633d.png)

**Comment by martindurant** [comment]
Yes, that line is there, but the release [does include](https://github.com/intake/filesystem_spec/blob/2021.04.0/fsspec/utils.py#L425) the necessary function to import

**Comment by martindurant** [comment]
OK, no worries
